packages/web/app/api/TamanuApi.jsx

```diff
@@ -174,8 +192,9 @@ export class TamanuApi extends ApiClient {
 
   async login(email, password) {
     const output = await super.login(email, password);
-    const { localisation, server, availableFacilities, permissions, role } = output;
+    const { localisation, server, availableFacilities, permissions, role, token } = output;
     saveToLocalStorage({
+      token,
```

Comment:
need to save token to local storage for session restore functionality to work

Reply:
are you sure? `super.login` should call `setToken`

Reply:
Yep in api-client's TamanuApi, SetToken just sets the tokens on the object and there are no interactions at all with localStorage.

packages/api-client/src/TamanuApi.js
````
...
  setToken(token, refreshToken = null) {
    this.#authToken = token;
    this.#refreshToken = refreshToken;
  }

Reply:
Just to clarify, when I say "session restore" above I mean the "restoreSession" method which runs on web app start and restores state from local storage.

Reply:
no, but, when super.login calls `this.setToken`, in the context of the the web's package TamanuApi, _that_ calls web's `TamanuApi.setToken`, which in turn calls api-client's `TamanuApi.setToken`.

```js
class Foo {
  setToken() {
    console.log('Foo.setToken')
  }

  login() {
    console.log('Foo.login')
    this.setToken()
  }
}

class Bar extends Foo {
  setToken() {
    super.setToken()
    console.log('Bar.setToken')
  }

  login() {
    super.login()
    console.log('Bar.login')
  }
}

(new Bar).login()

/* Prints:
Foo.login 
Foo.setToken 
Bar.setToken 
Bar.login 
*/

Reply:
oh yes you are totally right!